astropy/modeling/rotations.py

```diff
@@ -187,9 +187,6 @@ def evaluate(cls, x, y, angle):
         single coordinate pair.
         """
 
-        if x.shape != y.shape:
-            raise ValueError("Expected input arrays to have the same shape")
```

Comment:
Agreed. Is there a chance this will be fixed for 1.0?

The question is would removing this check from `evaluate` have any adverse effect on any other way the model is used? I think not, because the code would fail a few lines further down if the inputs have different shape. So my suggestion is to remove the check in 1.0 and open an issue for this problem.

If you disagree, feel free to put it back in.